BeerOverflowWindowsApp/MapControl.xaml.cs

```diff
@@ -47,19 +52,20 @@ private void MapWithPushpins_MouseDoubleClick(object sender, MouseButtonEventArg
 
         private void ChangeMapMode_Click(object sender, RoutedEventArgs e)
         {
-            if (Map.Mode.ToString() == "Microsoft.Maps.MapControl.WPF.RoadMode")
+            switch (Map.Mode.ToString())
             {
-                Map.Mode = new AerialMode(true);
-            }
-            else if (Map.Mode.ToString() == "Microsoft.Maps.MapControl.WPF.AerialMode")
-            {
-                Map.Mode = new RoadMode();
+                case "Microsoft.Maps.MapControl.WPF.RoadMode":
+                    Map.Mode = new AerialMode(true);
+                    break;
+                case "Microsoft.Maps.MapControl.WPF.AerialMode":
```

Comment:
Can you add a default case, just in case?

Reply:
Done. I guess we'll think later about possible case in here